wordpress-mobile/WordPress-Android
# Stats Views widget setup

This PR introduces classes necessary for the Views widget. I've also added the configure activity (it's not fully functional yet). Most of the stuff here doesn't need to be reviewed because it will be completely rewritten anyway but I didn't want to remove the classes since it took a lot of effort to make everything work. The Widgets API is dreadful. 

To test:
* Go to your Home and try to add a Widget
* Scroll to the Wordpress and add the new widget
* You see a new configure activity
* The Site and Color button don't work yet (TODO crashes the app if you click on them)
* The "Add Widget" button is disabled
